client/shipping-label/views/purchase/steps/packages/package-info.js

```diff
@@ -81,22 +93,26 @@ const PackageInfo = ( { packageId, selected, all, flatRateGroups, unpacked, dime
 		} );
 
 		return (
-			<FormSelect onChange={ packageOptionChange } value={ pckg.box_id }>
-				{ _.map( groups, ( group, groupId ) => {
-					if ( _.isEmpty( group.definitions ) ) {
-						return null;
-					}
-
-					return <optgroup label={ group.title } key={ groupId }>
-						{ _.map( group.definitions, renderPackageOption ) }
-					</optgroup>;
-				} ) }
-			</FormSelect>
+			<div>
+				<div className="wcc-package-items-header">
+					<FormLegend className="wcc-package-item__name">{ __( 'Box Package for Multiple Items' ) }</FormLegend>
```

Comment:
I think "Package" or "Package type" is simpler and more accurate here. This may not be a box, and it may not contain multiple items.